doc/user_guide/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
+SPHINXPROJ    = CoderDojoMobileToolbox
+SOURCEDIR     = source
+BUILDDIR      = build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile
+
+# Catch-all target: route all unknown targets to Sphinx using the new
+# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
+%: Makefile
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```

Comment:
Please add a new line at end of file

Reply:
Sandro, doc contains lot of PEP-8 non-compliant code... do we also have to correctly style that? Same for django migrations... they're automatically generated, style compliance is good for code managed by humans, not for everything :-)

Reply:
This is not pep-8, this is POSIX:

Because that’s how the POSIX standard defines a line:

3.206 Line
A sequence of zero or more non- <newline> characters plus a terminating <newline> character.

In this case your line is not terminated.
There've been lot of discussions about this on internet, referencing one just to get a general idea: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
@sandrobonazzola I fixed the missing newline, can you merge & close the PR?